### ISSUE TITLE
Autosave doesn't work anymore after we got 504 response on autosave

### ISSUE THREAD

**Issue by matchish** [nextcloud/deck#2175]
**Describe the bug**
If we get 504 response during autosave description, then autosave doesn't work until we reload the page

**Expected behavior**
Autosave have to work regardless of previous autosave request status

<details>
<summary>Server details</summary>

**Nextcloud version:** 18.0.4

**List of activated apps:**

Enabled:
  - accessibility: 1.4.0
  - bruteforcesettings: 1.5.0
  - calendar: 2.0.2
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - contacts: 3.2.0
  - dav: 1.14.0
  - deck: 1.0.5
  - federatedfilesharing: 1.8.0
  - federation: 1.8.0
  - files: 1.13.1
  - files_sharing: 1.10.1
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - mail: 1.1.3
  - oauth2: 1.6.0
  - onlyoffice: 4.1.4
  - password_policy: 1.9.0
  - provisioning_api: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - spreed: 8.0.8
  - systemtags: 1.8.0
  - theming: 1.9.0
  - twofactor_backupcodes: 1.7.0
  - updatenotification: 1.8.0
  - workflowengine: 2.0.0
Disabled:
  - admin_audit
  - defaultgroup
  - encryption
  - files_external
  - occweb
  - testing
  - user_ldap

</details>

**Comment by juliushaertl** [comment]
Please fill out the complete issue template especially the server and browser console logs.

**Comment by juliushaertl** [comment]
Hm a 504 is actually a gateway timeout. Do you have anything in your webserver logs?

**Comment by matchish** [comment]
You're right 504 was my server issue but I believe it should not break deck frontend. The issue is about autosave is broken after we have 504 response on frontend. Not about server. For example if you have cloudflare between server and browser you can get 504 and it will be not related to the server.

**Comment by juliushaertl** [comment]
Yes, we should of course properly catch that. I was just thinking that the saving is also failing on the deck side.

**Comment by matchish** [comment]
I see) Deck on server is fine. Infrastructure issue
